packages/web-components/src/components/va-modal/va-modal.tsx

```diff
@@ -132,6 +132,11 @@ export class VaModal {
    */
   @Prop({ reflect: true }) visible?: boolean = false;
 
+  /**
+   * DOM-nodes that should not be hidden from screen readers.
```

Comment:
Can we change the documentation so that people don't mistakenly think that they need to do something? We're basically adding this prop for ourselves to use, and _maybe_ there will be situations when someone else needs to use it. Something like this might work, but feel free to wordsmith it however you like:

> Additional DOM-nodes that should not be hidden from screen readers. Useful when an open modal shouldn't hide all content behind the overlay.